packages/sitecore-jss-angular/src/components/form.component.ts

```diff
@@ -136,6 +138,32 @@ export class FormComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Subscribes to the form submit event and sends data to CloudSDK.
+   */
+  subscribeToFormSubmitEvent() {
```

Comment:
strictly speaking, it's not only the submit event - more like 'engage' event

Reply:
makes sense.